foxglove/cmd/utils.go

```diff
@@ -1,17 +1,83 @@
 package cmd
 
 import (
+	"bytes"
 	"encoding/csv"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
 	"os"
 
 	"github.com/foxglove/foxglove-cli/foxglove/console"
+	"github.com/foxglove/mcap/go/mcap"
 	"github.com/olekukonko/tablewriter"
 	"github.com/spf13/cobra"
 )
 
+var ErrTruncatedMCAP = errors.New("truncated mcap file")
+var ErrInvalidInput = errors.New("supply a valid bag or mcap file")
```

Comment:
Just a thought... "supply a valid bag..." could be more descriptive based on context. I like that the truncated error tells me what's wrong with the file. (I do like the idea of a prescriptive action to take in the latter, but only if it's fairly specific.)

Reply:
updated that to be more specific